exception_reports/logs.py

```diff
@@ -32,7 +32,7 @@ def filter(self, record):
             try:
                 record.data["error_report"] = create_exception_report(exc_type, exc_value, tb, self.output_format, self.storage_backend)
             except Exception as e:
-                logger.warning(f"Error generating exception report {repr(e)}")  # noqa
+                logger.error(f"Error generating exception report {repr(e)}", exc_info=True)  # noqa
```

Comment:
The reason to not is that this could create an infinite recursion situation. Clearly only doing a warning is great either, but this might be making things worse.

Reply:
Sorry I didn't really explain.  One of the functionalities of the library is to create exception reports for error level logs so this would be calling itself.